src/psyclone/tests/nemo/transformations/profiling/nemo_profile_test.py

```diff
@@ -229,24 +251,26 @@ def test_profile_inside_if2(parser):
 def test_profile_single_line_if(parser):
     ''' Test that we can put a single-line if statement inside a
     profiling region. '''
-    psy, schedule = get_nemo_schedule(parser,
-                                      "subroutine one_line_if_test()\n"
-                                      "use kind_mod, only: wp\n"
-                                      "real :: sto_tmp2(jpj)\n"
-                                      "logical :: do_this = .true.\n"
-                                      "if(do_this) write(*,*) sto_tmp2(ji)\n"
-                                      "end subroutine one_line_if_test\n")
+    psy, schedule = get_nemo_schedule(
+        parser,
+        "subroutine one_line_if_test()\n"
+        "use kind_mod, only: wp\n"
+        "integer :: ji, jpj\n"
+        "real :: sto_tmp2(jpj)\n"
+        "logical, parameter :: do_this = .true.\n"
+        "if(do_this) write(*,*) sto_tmp2(ji)\n"
+        "end subroutine one_line_if_test\n")
     # Check that we refuse to attempt to split the body of the If from
     # its parent (as it is a one-line statement). This limitation will
     # be removed once we use the PSyIR Fortran backend in the NEMO API
     # (as opposed to manipulating the fparser2 parse tree).
     # TODO #435
     with pytest.raises(TransformationError) as err:
-        schedule, _ = PTRANS.apply(schedule[0].if_body)
+        _, _ = PTRANS.apply(schedule[0].if_body)
     assert "single-line if statement" in str(err.value)
     # But we should be able to put the whole If statement in a profiling
     # region...
-    schedule, _ = PTRANS.apply(schedule[0])
+    _, _ = PTRANS.apply(schedule[0])
```

Comment:
You can remove the underscores here and above if you like.

Reply:
Good point, done.